Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Services/NMDataServiceTest.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+using FakeItEasy;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using NUnit.Framework;
+using UKHO.FileShareClient.Models;
+using UKHO.MaritimeSafetyInformation.Common;
+using UKHO.MaritimeSafetyInformation.Common.Configuration;
+using UKHO.MaritimeSafetyInformation.Common.Models;
+using UKHO.MaritimeSafetyInformation.Web.Services;
+using UKHO.MaritimeSafetyInformation.Web.Services.Interfaces;
+
+namespace UKHO.MaritimeSafetyInformation.Web.UnitTests.Services
+{
+    [TestFixture]
+    public class NMDataServiceTest
+    {
+        private IFileShareService _fakefileShareService;
+        private IOptions<FileShareServiceConfiguration> _fileShareServiceConfig;
```

Comment:
Remove this variable _fileShareServiceConfig as its unused.

Reply:
Variable removed